content/4_Gold/Tree_Euler.mdx

```diff
@@ -715,6 +715,81 @@ preprocessing and query in $\mathcal{O}(1)$ time.
 
 <FocusProblem problem="rmqSample" />
 
+The following is an example implementation of a sparse table and code that
+answers LCA queries. Build time is $\mathcal{O}(N\log N)$, and queries are
+$\mathcal{O}(1)$.
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+template <class T> struct RMQ {
```

Comment:
huh weird what''s w/ `class` instead of `typename`
i'm just curious bc i always use the latter

Reply:
oh i normally use class b/c its shorter to type, but typename makes more sense in this case